src/Admin/Users/ReportedUsers/ReportedUsersAdmin.php

```diff
@@ -19,6 +22,46 @@ class ReportedUsersAdmin extends AbstractAdmin
    */
   protected $baseRoutePattern = 'reported_users';
 
+  protected function configureQuery(ProxyQueryInterface $query): ProxyQueryInterface
+  {
+    /** @var ProxyQuery $query */
+    $query = parent::configureQuery($query);
+
+    $qb = $query->getQueryBuilder();
+
+    $rootAlias = $qb->getRootAliases()[0];
+    $parameters = $this->getFilterParameters();
+
+    if ('getReportedCommentsCount' === $parameters['_sort_by']) {
+      $qb
```

Comment:
These `qb`s had an extra `from` call. I guess it worked for a while until some dependency update broke it.